<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Bernhard Beschow</author>
 <class>DependsUIBase</class>
 <widget class="QMainWindow" name="DependsUIBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DPWalker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="dpwalker.qrc">
    <normaloff>:/icons/depends.png</normaloff>:/icons/depends.png</iconset>
  </property>
  <property name="opaqueMoving" stdset="0">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QVBoxLayout">
    <item>
     <widget class="QSplitter" name="splitter1">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="opaqueResize">
        <bool>true</bool>
       </property>
       <property name="childrenCollapsible">
        <bool>true</bool>
       </property>
       <widget class="QTreeWidget" name="m_pTreeSharedObjects">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>50</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="rootIsDecorated">
         <bool>true</bool>
        </property>
        <property name="showSortIndicator" stdset="0">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>Imported Libraries</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Path</string>
         </property>
        </column>
       </widget>
       <widget class="QSplitter" name="splitter3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>50</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Import Symbols</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QTreeWidget" name="m_pListImports">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>true</bool>
            </property>
            <property name="showSortIndicator" stdset="0">
             <bool>true</bool>
            </property>
            <attribute name="headerShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Export Symbols</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QTreeWidget" name="m_pListExports">
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>true</bool>
            </property>
            <property name="showSortIndicator" stdset="0">
             <bool>true</bool>
            </property>
            <attribute name="headerCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="headerHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="headerShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>VAdr</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Binding Type</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Weak</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Constructor</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Warning</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Indirect</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Debug</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Section</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Size</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Library</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>688</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="fileOpenAction"/>
    <addaction name="separator"/>
    <addaction name="fileExitAction"/>
   </widget>
   <addaction name="fileMenu"/>
  </widget>
  <action name="fileOpenAction">
   <property name="icon">
    <iconset resource="dpwalker.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="iconText">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="name" stdset="0">
    <cstring>fileOpenAction</cstring>
   </property>
  </action>
  <action name="fileExitAction">
   <property name="icon">
    <iconset resource="dpwalker.qrc">
     <normaloff>:/icons/close.png</normaloff>:/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="iconText">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="name" stdset="0">
    <cstring>fileExitAction</cstring>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="dpwalker.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fileOpenAction</sender>
   <signal>triggered()</signal>
   <receiver>DependsUIBase</receiver>
   <slot>fileOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileExitAction</sender>
   <signal>triggered()</signal>
   <receiver>DependsUIBase</receiver>
   <slot>fileExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pTreeSharedObjects</sender>
   <signal>itemClicked(QTreeWidgetItem*,int)</signal>
   <receiver>DependsUIBase</receiver>
   <slot>dependencySelected(QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fileOpen()</slot>
  <slot>fileExit()</slot>
  <slot>dependencySelected(QTreeWidgetItem*)</slot>
 </slots>
</ui>
